alacritty/src/config/ui_config.rs

```diff
@@ -25,7 +25,7 @@ use crate::config::window::WindowConfig;
 
 /// Regex used for the default URL hint.
 #[rustfmt::skip]
-const URL_REGEX: &str = "(ipfs:|ipns:|magnet:|mailto:|gemini:|gopher:|https:|http:|news:|file:|git:|ssh:|ftp:)\
+const URL_REGEX: &str = "(ipfs:|ipns:|magnet:|mailto:|gemini://|gopher://|https://|http://|news:|file://|git://|ssh:|ftp://)\
```

Comment:
I'm not sure `file://` should be enforced. I wouldn't be surprised to see `file:/home/user/whatever` in the wild and it generally works just fine.

Reply:
Sure, done.